InferenceFunctionApp/score/predict.py

```diff
@@ -0,0 +1,85 @@
+import base64
+from io import BytesIO
+import json
+import os
+import logging
+import onnxruntime
+import numpy as np
+from torchvision.transforms import transforms
+from scipy.special import softmax
+from PIL import Image
+
+
+def load_labels(path):
+    """Loads labels from a json file.
+
+    Args:
+        path (str): Path to labejson file
+
+    Returns:
+        ndarray: Array of labels
+    """
+    with open(path, encoding="utf-8") as file:
+        data = json.load(file)
+    return np.asarray(data)
+
+
+d = os.path.dirname(os.path.abspath(__file__))
+logging.info("Loading model from: %s", d)
+modelfile = os.path.join(d, "classificationModel.onnx")
+labelfile = os.path.join(d, "labels.json")
+
+session = onnxruntime.InferenceSession(modelfile, None)
+input_name = session.get_inputs()[0].name
+
+labels = load_labels(labelfile)
+
+
+def preprocess(base64_image):
+    """Returns a tensor of the image after preprocessing.
+
+    Args:
+        base64_image: Image in base64 format
+
+    Returns:
+        numpy.ndarray: Transformed image tensor in numpy array format
+    """
+    transformation = transforms.Compose(
+        [
+            transforms.Resize(256),
+            transforms.CenterCrop(224),
```

Comment:
nie powinny być tu dwie takie same wartości?

Te wartości w ogóle zależą od tego jak trenowaliśmy model - da się to jakoś wyczytać z modelu?
Jeżeli nie, to na góre pliku jako stałe.

Reply:
W sumie to nie jestem pewien. Zrobiłem tak jak mamy w inferencji w skrypcie do trenowania. Mogę wrzucić zadanie na zbadanie metod auguementacji i przy okazji pobawić się tym trochę.